packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -53,7 +64,7 @@ class DiffPaneController extends DisposableController {
       defaultName: selectedIsolateName ?? '<isolate-not-detected>',
     );
 
-    await _addSnapshot(_heapGraphLoader, item);
+    await _addSnapshot(loader, item);
```

Comment:
nit: no need to store loader in a local variable. Just null assert it here `loader!`